node/src/accountant/payment_adjuster/logging_and_diagnostics/diagnostics.rs

```diff
@@ -100,7 +100,7 @@ pub mod ordinary_diagnostic_functions {
     use thousands::Separable;
     use web3::types::Address;
 
-    pub fn thriving_competitor_found_diagnostics(
+    pub fn diagnostics_for_accounts_above_disqualification_limit(
```

Comment:
Does it mean that these accounts are disqualified or qualified? I'd suggest you either add it as a comment, or possibly rename the fn to make it clear.